types/electra/attestation.yaml

```diff
@@ -0,0 +1,33 @@
+Electra:
+  IndexedAttestation:
+    type: object
+    description: "The [`IndexedAttestation`](https://github.com/ethereum/consensus-specs/blob/dev/specs/electra/beacon-chain.md#indexedattestation) object from the CL spec."
```

Comment:
i think we'd be better off locking it into a spec version, even if it's alpha... it'll mean we need to update it at some point but even currently we'll need to move it off of dev...

Reply:
done, used the alpha version for now

Reply:
great - thats better - we'll just need to periodically update it.